packages/babel-helper-define-polyfill-provider/src/index.js

```diff
@@ -60,6 +60,19 @@ function resolveOptions<Options>(
     ...providerOptions
   } = options;
 
+  if (isEmpty(options)) {
+    const example = {
+      plugins: ["babel-plugin-polyfill-corejs3", { method: "usage-pure" }],
+    };
+
+    throw new Error(
+      `This plugin requires options, for example: \n` +
+        `${JSON.stringify(example, null, 2)} \n` +
+        `See more options at ` +
+        `https://github.com/babel/babel-polyfills/blob/main/docs/usage.md`,
+    );
```

Comment:
Also, I'd prefer to write a generic `<plugin name>` instead of `babel-plugin-polyfill-corejs3`

Reply:
Ah that does look nicer (and cleaner code-wise), will make that change shortly!

> I'd prefer to write a generic `<plugin name>` instead of `babel-plugin-polyfill-corejs3`

Can change to that too. I assumed by "example" #17 meant a like real, usable config, so I went with what is likely the most common config/polyfill. But yea given this is error code in the helper, probably best to not mention a specific plugin name that could be different from the given plugin; didn't even think of `<plugin name>` as an option!

Reply:
Changes made